### PR TITLE
Adds tests for BoardGame Poro and GameFacade

### PR DESCRIPTION
**Pull request by MarchandMD**
### Why is this PR being made?  

To add the necessary Facade and PORO to process incoming data from 3rd party API

### How is this accomlished?  

By passing the incoming service data to the Facade, an Array is created containing `BoardGame` poro objects

### CODE REVIEW

File: app/poros/board_game.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+              :max_players,
+              :min_age,
+              :min_playtime,
+              :max_playtime,
+              :description,
+              :image_url,
+              :thumb_url
+
+  def initialize(data)
+    @id = data[:id]
```
Comment (DarbySmith) [inline app/poros/board_game.rb:16]: I believe this should be board_game_atlas_id rather than id

File: app/poros/board_game.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class BoardGame
+  attr_reader :id,
```
Comment (DarbySmith) [inline app/poros/board_game.rb:2]: Same comment as below

**Review by DarbySmith — COMMENTED** [review]
I think the id attribute should be changed to board_game_atlas_id to match the schema design and responses we built for the FE

**Comment by MarchandMD** [comment]
This is addressed in PR #24

**Review by bkeener7 — APPROVED** [review]

**Pull request merged**